src/Views/Form/Templates/Sequoia/assets/css/form.scss

```diff
@@ -313,6 +313,8 @@ p {
 	}
 
 	.description + .advance-btn {
+		margin-top: 6px !important;
```

Comment:
Is `!important` required to achieve this? If so, go ahead and merge but when I see this I sort of cringe a little. However, I'm not going to put up a big fuss about it.

Reply:
Henry informed me this is required.